shell/edit/provisioning.cattle.io.cluster/tabs/Networking.vue

```diff
@@ -2,9 +2,9 @@
 import { LabeledInput } from '@components/Form/LabeledInput';
 import { Banner } from '@components/Banner';
 import { Checkbox } from '@components/Form/Checkbox';
-import { _EDIT } from '@shell/config/query-params';
+import { _EDIT, _VIEW } from '@shell/config/query-params';
 import ArrayList from '@shell/components/form/ArrayList';
-import ACE from './ACE';
+import ACE from '@shell/edit/provisioning.cattle.io.cluster/ACE';
```

Comment:
same as per etcd and S3Config,  this ACE can move in to it's parent components folder